File: Java/NoorTeck/Day5Homework/Grade.java
/**
 * Grade
 */
public class Grade {

    public static void main(String[] args) {

        int score = 45;
        char grade = 'A';

        if ( score >= 90 ) {
            grade = 'A';

        } else if ( score >= 80 )  {
            grade = 'B';

        } else if ( score >= 70 ) {
            grade = 'C';

        } else if ( score >= 60 ) {
            grade = 'D';
             
        } else {
            grade = 'F';
        }

        System.out.println( score );
        System.out.println( grade );
    }
}

File: Data_Types_And_Control_Flow/3.grade_problem/grade.py
mark = int(input("Tell me how many points you have and I will tell you the grade :"))
if mark > 100:
    print('Invalid number please check the number and Enter again !')
elif  mark >= 90 and mark <= 100:
    print('You got an A')
elif mark >=70 and mark <= 89:
    print('You have a B')
elif mark >=50 and mark <= 69:
    print('You have a C')
elif mark >=35 and mark <= 49:
    print('You have a D')
else:
    print('You FAILED')
